### DISCUSSION TITLE
Hardware acceleration for AMD 7840hp

### DISCUSSION THREAD

**Discussion by groyee** [blakeblackshear/frigate#10514] [Hardware Acceleration Support] [answered]
### Describe the problem you are having

Hi

I have Beelink GTR7 with AMD Ryzen 7840HS. What is the right hwaccel_args preset for that CPU/GPU?

Thanks

### Version

0.13.2

### Frigate config file

```yaml
none
```

### docker-compose file or Docker CLI command

```yaml
none
```

### Relevant log output

```shell
none
```

### FFprobe output from your camera

```shell
none
```

### Operating system

HassOS

### Install method

HassOS Addon

### Network connection

Wired

### Camera make and model

tplink c110

**Comment by NickM-27** [comment, marked as answer]
https://docs.frigate.video/configuration/hardware_acceleration#amdati-gpus-radeon-hd-2000-and-newer-gpus-via-libva-mesa-driver

**Reply by groyee** [reply]
Thanks. I saw this but wasn't sure. I guess from your reply the answer is: `preset-vaapi`

Thanks!

**Reply by NickM-27** [reply]
You'll need to set the env variable as  well

**Comment by Honusnap** [comment]
Hello, 
To add to that, is ther any posibilities to use the AI part of those processors for scene detection ? Kinf like a coral would do ?
